platform/accessibility/vsp-accessibility-rules-of-engagement.md

```diff
@@ -0,0 +1,93 @@
+
+For medium to large projects, there will be five accessibility touch points. Teams can also reach out to accessibility specialists for spot checks, small-group consults, or code reviews any time.
+
+1. **Project Kickoff**<br/>
+
+   <br/>
+
+   Tips for a successful project:
+
+   * Read through the [508 checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-checklist.md). This checklist will give you a good overview of accessibility requirements and tools that will help you in early-stage testing.
+
+   * Read through the [508, accessibility, and inclusive design best practices](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md#color-tests). This goes into greater detail on the areas your product will be evaluated for accessibility and inclusivity.
+
+   * Reach out to your accessibility specialist(s) if you have questions after the product kickoff meeting. The sooner your team is comfortable with these accessibility guidelines, the more successful your project will be.
+
+2. **Design Intent Collaboration** — *Integrated with [VSP design intent check-in](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/design/working-with-platform-design-team.md#design-intent-check-in)*<br/><br/>Tips for designing accessible solutions:
+
+   * Involve the accessibility specialist in discussions about needs, priorities, concepts, research planning, and direction.
+
+   * Include the accessibility specialist in formal and informal reviews to ensure inclusive design is woven through the efforts. Accessibility specialists can help identify challenges with user flows, color contrast, and color blindness at this stage of the software development lifecycle (SDLC).
+
+   * Consider the following elements while designing user experiences:
+     * [Performance budgets](https://addyosmani.com/blog/performance-budgets/)
+     * Mobile-first design 
+     * Mobile-only interactions
+     * Touch interactions
+     * Spacing of interactive items – for example, is the spacing sufficient to ensure people don't accidentally select something other than what they meant to?
+     * Laptop/desktop design
+     * Semantic heading order
+     * Cognitive impact – in the words of [Steve Krug](http://sensible.com/), *don't make me think!* Design should be clear, not overwhelming.
+
+3. **Usability Testing Prep** — *Integrated with [VSP pre-usability design review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/design/working-with-platform-design-team.md#design-reviews-1)*<br/><br/>The accessibility specialist will provide more specific feedback, including:
+
+   * Relevant design system components and patterns
+
+   * Feedback on recruiting diverse research participants
```

Comment:
I worry about providing feedback on recruiting diverse research participants only in this checkpoint. This is likely to be valuable info to provide for consideration during the design intent collaboration, as it can take significant time to recruit participants with specific parameters.